packages/core/core/src/utils/logger/defaultLogger.ts

```diff
@@ -1,10 +1,10 @@
 /* istanbul ignore file */
 
 const defaultLogger = {
-  debug: (message: any, ...args) => console.log('[VSF][debug]', message, ...args),
-  info: (message: any, ...args) => console.log('[VSF][info]', message, ...args),
-  warn: (message: any, ...args) => console.log('[VSF][warn]', message, ...args),
-  error: (message: any, ...args) => console.log('[VSF][error]', message, ...args)
+  debug: (message: any, ...args) => console.debug('[VSF][debug]', message, ...args),
```

Comment:
those calls don't work on Node, so please keep `log`

Reply:
I tested it and it worked fine for me. Node docs suggest that it always supported these methods.